README.md

```diff
@@ -18,3 +18,16 @@ snakemake -j1 --use-conda
 ```
 
 Output will show up in the `results/` directory
+
+
+## Supported Models
+
+We have pre-trained ENCODE-rE2G on certain model types. You can find them in the `models` directory.
+Each model must have the following:
+1. model pickle file
+2. feature table config 
```

Comment:
@mayasheth seems like we also need a feature table config file for each model?

Reply:
I'm going to assume the feature table config that we use for dhs_intact_hic and dhs_megamap are the same?

Reply:
Yes: feature table/config determines what features are required for a specific model and what they're called, which is needed to create final feature table. also ensures the names of the features in the trained model align with the feature tables.

Yes, feature tables for dhs_intact_hic and dhs_megamap are the same since we are pulling from the same ABC contact column and we decided to use the same set of features.